test/functional/bulk_tests.js

```diff
@@ -937,68 +937,59 @@ describe('Bulk', function() {
     }
   });
 
-  it(
-    'should provide descriptive error message for Unordered Batch with duplicate key errors on updates',
-    {
-      metadata: { requires: { topology: ['single', 'replicaset', 'ssl', 'heap', 'wiredtiger'] } },
-
-      test: function(done) {
-        const configuration = this.configuration;
-        var client = configuration.newClient(configuration.writeConcernMax(), {
-          poolSize: 1
-        });
-
-        client.connect(function(err, client) {
-          var db = client.db(configuration.db);
-          var col = db.collection('err_batch_write_unordered_ops_legacy_6');
-
-          // Write concern
-          var writeConcern = configuration.writeConcernMax();
-          writeConcern.unique = true;
-          writeConcern.sparse = false;
+  it('should provide descriptive error message for unordered batch with duplicate key errors on inserts', function(done) {
+    const configuration = this.configuration;
+    const client = configuration.newClient(configuration.writeConcernMax(), {
+      poolSize: 1
+    });
 
-          // Add unique index on b field causing all updates to fail
-          col.ensureIndex({ b: 1 }, writeConcern, function(err) {
-            expect(err).to.not.exist;
+    client.connect(function(err, client) {
```

Comment:
For all callbacks in this test, let's use `(err, client) => {` notation instead of `function(err, client) {`.